docs/quantum-consciousness.org

```diff
@@ -0,0 +1,197 @@
+
+Without getting into the sentimental and inflammatory...  I should beg of you to take a drink and a snack, this is going to be a long and bumpy ride.
+* The nature of consciousness
+
+The nature of consciousness is an elusive problem that despite being clearly defined within the reach of regular, common sense, seems to not be possible to neatly package into byte-sized analytical chunks and have science give us a clear answer.  This should not surprise you.  Cognising cognition is a self-referential paradox, the kind that bought down mathematics more than once.  What should surprise you (but doesn't) is that the question of "free will" is lumped together, despite being orthogonal and largely a semantic argument: if you believe in superdeterminism, free will doesn't just "not exist", it's ill-defined[fn:1]!
+
+We all know what we mean when a clone in science fiction would say is this the "*you* - you, or the other - you".  Unfrtounately, that is where the clarity of the problem ends.  Some take it into the mystical domain, arguing that consciousness is your soul.  Some take it into the opposite, and equally stupid direction: you have no consciousness, it's all an elaborate illusion.  What I find most painful, is that while the former camp is clearly labelled as a bunch of crackpots that should not be taken seriously, the second camp[fn:2], is considered the /rational point of view/.  The truth of the matter is that we don't know.  Anyone who claims otherwise, have chosen to ignore more than half of the problem, and have thus come up with a meaningless answer.
+
+With that said, there is a surprising resurgence in interest related to the question of what does it mean to understand somehing.  What does it mean to know something.  Is this connected to consciousness, or something that our bodies can do autonomously.  Chomsky's room is a good example of this resurgence, despite the mental experiment being proposed a good few years back, transformer-based next-token prediction machines that show a clear propensity to mimic intelligent discourse have sparked a resurgence in that discussion.  Can machines think?  If machines can think now, can a future machine be built that can?  Does the fact that we are built out of meat and machines are built out of something else matter for the ability to understand and synthesise cognition?  Is symbolic reasoning even necessary to call something conscious?
```

Comment:
> Is symbolic reasoning even necessary to call something conscious?

That's a great topic to raise because "symbolic reasoning" goes partially towards the "memorizing" part: we're not just remembering facts; we process said facts and find some links, for instance, **causal**, **chronological**, **spatial**, **thematic**, etc. So, "symbolic reasoning" is not just about using symbols, not just about learning ones from others, it's all about introducing new abstract symbols and memorizing them with a given time interval.

Plus, **while neurons work as detectors**, the transformer models we interact with so far **display a curious property**: I don't remember them **requesting clarifications** when I ask whatever questions that come to mind, however ridiculous they are. Even if this architecture displays symbolic reasoning, it's all about the levels of abstraction and the relationships between symbols, as "[Scaling Monosemanticity: Extracting Interpretable Features from Claude 3 Sonnet](https://transformer-circuits.pub/2024/scaling-monosemanticity/)" easily demonstrates.

Now, just to confirm my point, let's conduct a silly test: I'm using a Bluetooth mouse, so it'll be a question about Harald Bluetooth. And I love otters, so let's ask [GPT4-o](https://chatgpt.com/share/2ba7e3a3-8d33-4847-a93f-d373d4f271e8) about king's pet otter to validate whether it's checking its confidence about symbols or not!

> According to legend, Harald Bluetooth had a pet otter named "Sot" (sometimes spelled "Sote").

> Sot was not an ordinary otter but was considered to have almost mystical or supernatural qualities.

> The legend suggests that Harald was very fond of Sot and the otter was often seen by his side.

> There is no concrete historical evidence to confirm the existence of Harald Bluetooth’s pet otter. The story is mostly passed down through oral traditions and later medieval texts.

> Harald Bluetooth, a king of Denmark and Norway in the 10th century, is a significant historical figure known for his role in unifying Denmark and converting the Danes to Christianity. However, the story of Harald Bluetooth's pet otter is more of a myth or legend rather than a well-documented historical fact.